### ISSUE TITLE
Copy over Proper Motion catalogues and setup

### ISSUE THREAD

**Issue by wfastrononomer** [wfau/ScienceArchives#520]
Copy over VMC PM catalogues

Setup ExternalProduct directory 

Run convertTeamCats code

Done for LMC. But not properly linked to PSF.

**Comment by tms-epcc** [comment]
08/MAY/24
@wfastrononomer reported
- LMC has been added in
- need to link this to the PSF data

**Comment by tms-epcc** [comment]
05/JUN/24
- have a first version but 
- still to add epoch column
- and  need to deal with bridge and stream data

**Comment by tms-epcc** [comment]
11/JUL/24
- still to add epoch into proper motion catalogues

**Comment by wfastrononomer** [comment]
26/JUL/24 - Added in PM column. Still need SMC data.

**Comment by tms-epcc** [comment]
14/AUG/24
- still need SMC data

**Comment by wfastrononomer** [comment]
At the following link you find the SMC proper motion tables:  https://cloud.aip.de/index.php/s/BXajBFzccMFRY6a.

There is only one missing, which I am going to send you shortly. This is the one with the 47 Tuc cluster.

Let me know when you have downloaded them.

I got back to work this week and I am going to try to finalise the material for the data release.
Will keep you posted!

How is it going on your side?

I did tell ESO that we would be sending all of the data in September, let’s try our best and then see to which date we come up to.

**Comment by wfastrononomer** [comment]
This is actually not the tile with 47 Tuc, but a regular tile in the centre of the SMC.

For this tile we have more epochs that were obtained by Viktor, but we are currently looking at the shift of the tile pointing which may influence the proper motion.

I will be back to you about it. If we decide that the observations by Viktor cannot be combined with the VMC observations without introducing distortions then I will send you quickly the existing proper motion table for this tile.

Instead, if we are going to recalculate the proper motions using also Viktor’s observations we then need to wait for Sreepriya, the Master student who has done the calculations for all other tiles, to return from a visit to her home country. She will return in abotu 2 weeks.

**Comment by wfastrononomer** [comment]
Below is the link to download the proper motion table for tile SMC 5_4. 
This is just based on the original VMC data because the data collected by Viktor used a different tile pattern.
Let me know once you download it successfully.

**Comment by wfastrononomer** [comment]
No, it will not be replaced because of the different tile pattern used which introduces distortions when combined at the detector level.
This is the final one.

**Comment by wfastrononomer** [comment]
Got SMC apart from gap and missing STR_2_1

Have to merge SMC reference epoch into others.

**Comment by wfastrononomer** [comment]
Recreate from scratch now that there is an epoch column.

**Comment by wfastrononomer** [comment]
Final catalogue has 11934406 rows

**Comment by wfastrononomer** [comment]
Still need gap data ....

**Comment by wfastrononomer** [comment]
LMC 2_5 seems to have 3x the number of sources as in the input catalogue - why? 

LMC 6_1 is missing and LMC 6_10 seems to have as many sources as LMC 6_1 and 6_10 combined. 

Add some code (possibly a new column in ExternalProductCatalogue) that checks input data rows against completed table.

**Comment by tms-epcc** [comment]
16/OCT/24
- @wfastrononomer adding code to check to help identify the issues

**Comment by tms-epcc** [comment]
20/NOV/24
- needs a further check to confirm if done
